### PR TITLE
Reminder support for SpringActorClassFinder

### PR DESCRIPTION
**Pull request by johnou**

### CODE REVIEW

File: actors/providers/spring/src/main/java/com/ea/orbit/actors/providers/spring/SpringActorClassFinder.java [hunk 1]
```diff
@@ -75,7 +78,9 @@ public void initialize()
         {
             throw new NullPointerException("orbit.actors.basePackage must be set!");
         }
-        this.concreteImplementations = actorImplementationProvider.getActorImplementations(actorInterfaceProvider.getActorInterfaces());
+        Map<Class<?>, Class<?>> implementations = actorImplementationProvider.getActorImplementations(actorInterfaceProvider.getActorInterfaces());
+        implementations.put(IReminderController.class, ReminderController.class); // built in Orbit actor
```
Comment (johnou) [inline actors/providers/spring/src/main/java/com/ea/orbit/actors/providers/spring/SpringActorClassFinder.java]: maybe built-in Orbit system actors like ReminderController could be exposed somehow.

Comment (johnou) [inline actors/providers/spring/src/main/java/com/ea/orbit/actors/providers/spring/SpringActorClassFinder.java]: the latter makes more sense in my opinion.

Comment (DanielSperry) [inline actors/providers/spring/src/main/java/com/ea/orbit/actors/providers/spring/SpringActorClassFinder.java]: Indeed. The reminder controller initialization needs to be done from the nodes. 
Thanks.

**Comment by DanielSperry** [comment]
Hi @johnou could you check if https://github.com/electronicarts/orbit/commit/c6ac3b3c8a9cf528b8a044cc268f34725855bdd7 addresses this issue?

**Pull request merged**